packages/gatsby-plugin-manifest/package.json

```diff
@@ -32,7 +32,7 @@
   "license": "MIT",
   "main": "index.js",
   "peerDependencies": {
-    "gatsby": "^2.25.0"
+    "gatsby": "^2.4.0"
```

Comment:
Reverted this for the release branch for additional safety. This bump will go to the next minor.